src/templates/kit/screens/TokensScreen.js

```diff
@@ -395,13 +390,13 @@ function MemberField({
   )
 }
 
-function formatReviewFields(tokensData) {
+function formatReviewFields(screenData) {
   return [
     [
       'Token name & symbol',
-      `${tokensData.tokenName} (${tokensData.tokenSymbol})`,
+      `${screenData.tokenName} (${screenData.tokenSymbol})`,
     ],
-    ...tokensData.members.map(([account], i) => [
+    ...screenData.members.map(([account], i) => [
       `Tokenholder #${i + 1}`,
```

Comment:
Should this be 2 words? `Token holder #...`?

Reply:
No it was decided to use the term “Tokenholder” rather than “Token holder” from now on.

@dizzypaty @owisixseven @john-light @luisivan I’m not sure we have these rules written anywhere − should we start a document somewhere for content guidelines?